kong-mesh/kong-mesh-sidecar-scc-cni.yaml

```diff
@@ -0,0 +1,38 @@
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
+- NET_BIND_SERVICE
```

Comment:
In our previous testing we found excluding these linux capabilities did not work because the side is starting additional process in the container such as OPA

- SETGID
- SETUID
- FSETID

Reply:
This is indeed a good point. I didn't not test OPA actually. Let me test more scenarios. I was working on a task which drops as many capabilitie requirements as we can: https://github.com/kumahq/kuma/pull/9656
If these are required, I'll have to revert that work. 

Thank you very much.